src/main/java/io/split/android/client/events/SplitEventsManager.java

```diff
@@ -188,19 +217,32 @@ private void triggerEventsWhenAreAvailable() {
         }
     }
 
-    private void trigger(SplitEvent event) {
+    // MARK: Helper functions.
+    private boolean isTriggered(SplitEvent event) {
```

Comment:
do we need `isTriggered` when we already have `wasTriggered` ? 
I am not sure why you are using one method in some cases and the other one in others.